articles/custom-error-pages.md

```diff
@@ -0,0 +1,113 @@
+-	Support URL: the URL of your Support team's webpage.
+
+![](/media/articles/error-pages/error-page-settings.png)
+
+### Customized Error Pages
+
+If you choose to display a custom error page, you have two options:
+
+-	You may redirect the user to a custom error page;
+-	You may configure Auth0 to render a custom error page on your behalf (please note that this feature is only available via the Management API).
+
+#### Redirecting Users to a Custom Error Page
+
+To redirect users to a custom error page:
+
+1. On the Account Settings page, scroll down to the Error Pages section.
+2. Select the option "redirect users to your own error page."
+3. Provide the URL of the error page you would like your users to see.
+
+![](/media/articles/error-pages/redirect-error-page.png)
+
+## Customizing Error Pages via the Management API
+Instead of using the Management Portal, you may configure your error pages by making the appropriate PATCH call to the Management API.
+
+![](/media/articles/error-pages/patch-tenant-settings.png)
+
+Prior to beginning, please ensure that you are logged in to an account that is permitted to make changes to your Auth0 configuration.
+
+1. Navigate to the [APIv2 Explorer Page](/api/v2).
+2. Go to the [Update Tenant Settings](api/v2#!/Tenants/patch_settings) section of the [APIv2 Explorer Page](/api/v2).
+3. Under Scopes, click on "update:tenant_settings" to add the scope required for this particular endpoint to the [API token](api/v2/tokens).
+4. Populate the "body" field with the JSON snippet containing the changes you would like to make. Further details on what to include in this snippet is included below.
+
+To assist you in customizing the required JSON snippet, the "Show samples" link in the upper right corner of the API Explorer Window will display the following sample code:
+
+```
+{
+  "error_page": {
+    "html": "",
+    "show_log_link": false,
+    "url": "https://mycompany.org/error"
+  },
+  "friendly_name": "My Company",
+  "picture_url": "https://mycompany.org/logo.png",
+  "support_email": "support@mycompany.org",
+  "support_url": "https://mycompany.org/support"
+}
+```
+
+### The Auth0 Default Error Page
+Even if you choose to display the default Auth0 error page, you may customize the following fields:
+
+-	Friendly Name: a user-friendly version of your company's name;
+-	Logo URL: the path to the logo you want to show to users;
+-	Support Email: the email address for your Support team;
+-	Support URL: the URL of your Support team's webpage.
+
+The following JSON will set the appropriate fields when included in your call to the API:
+
+```
+"friendly_name": "My Company",
+"picture_url": "https://mycompany.org/logo.png",
+"support_email": "support@mycompany.org",
+"support_url": "https://mycompany.org/support"
+```
+
+### Customized Error Pages
+
+#### Redirecting Users to a Custom Error Page
+To redirect users to a custom error page, update the "url" field to point to the location of the error page:
```

Comment:
What is the API endpoint to send this to. This is a bit unclear, think about the cases where users are linked directly to a subsection of the doc. Each subsection should be usable independently or at least point to relevant documentation. It is a common use case to link users to a section of an article like: https://auth0.com/docs/connections/database/migrating#enable-automatic-migration

For any place that includes an API you should generally include the following:

METHOD === the HTTP Method (GET, POST, etc.)

```
METHOD https://url
```

Request Body:
```
{
   "foo": "baz"
}
```